src/openms_gui/include/OpenMS/VISUAL/LayerData.h

```diff
@@ -32,6 +32,8 @@
 // $Authors: Marc Sturm $
 // --------------------------------------------------------------------------
 
+#error No not use this header...
```

Comment:
?

Reply:
I left the old header and cpp in the source tree (for now), just for reference.
To make sure noone is using the header until its removed, I added the macro.

Reply:
I added a TODO here, just to make sure its not forgotten:
https://github.com/OpenMS/OpenMS/projects/39#card-47792498